src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -112,15 +125,9 @@ public CountingReadFilter makeReadFilter(){
         return defaultFilters;
     }
 
-    /**
-     * Get the information about how to downsample the reads. By default {@link DownsamplingMethod#NONE} is returned.
-     * Subclasses should override it to provide their own downsampling methods.
-     *
-     * @return the downsampling method for the reads
-     */
-    public DownsamplingMethod getDownsamplingMethod() {
-        // TODO: change when downsampling command line options are implemented
-        return DownsamplingMethod.NONE;
+    /** Returns the downsampling info using {@link #maxDepthPerSample} as target coverage. */
+    protected final LIBSDownsamplingInfo getDownsamplingInfo() {
+        return new LIBSDownsamplingInfo(maxDepthPerSample != 0, maxDepthPerSample);
```

Comment:
set the second argument to the `LIBSDownsamplingInfo` to -1 in the case where we're not downsampling, as per the method javadoc